From ff3652e0a34de0dc118cf85bb50725cacb9d0967 Mon Sep 17 00:00:00 2001
From: Fan Zhang <zhfan@google.com>
Date: Fri, 27 Jul 2018 16:16:43 -0700
Subject: [PATCH 42/71] Clean up task affinity.

If activity is launched through external intent, back button should just
go back to previous screen.

Change-Id: I5b2e57d2751f5ed7c17140d3eb8d54736b78053c
Merged-In: I5b2e57d2751f5ed7c17140d3eb8d54736b78053c
Bug: 112254258
Fixes: 111864355
Test: manual
---
 AndroidManifest.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 2c5f846511..39e049c400 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -731,7 +731,6 @@
             android:label="@string/zen_mode_settings_title"
             android:icon="@drawable/ic_notifications"
             android:exported="true"
-            android:taskAffinity="com.android.settings"
             android:parentActivityName="Settings">
             <intent-filter android:priority="1">
                 <action android:name="android.settings.ZEN_MODE_SETTINGS" />
@@ -933,7 +932,6 @@
             android:label="@string/night_display_title"
             android:enabled="@*android:bool/config_nightDisplayAvailable"
             android:icon="@drawable/ic_settings_night_display"
-            android:taskAffinity="com.android.settings"
             android:parentActivityName="Settings">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
@@ -2620,7 +2618,6 @@
         <activity
             android:name="Settings$ZenAccessSettingsActivity"
             android:label="@string/manage_zen_access_title"
-            android:taskAffinity="com.android.settings"
             android:parentActivityName="Settings">
             <intent-filter android:priority="1">
                 <action android:name="android.settings.NOTIFICATION_POLICY_ACCESS_SETTINGS" />
-- 
2.17.1

